### ISSUE TITLE
Is PR welcomed?

### ISSUE THREAD

**Issue by albinsun** [tkhoa2711/vim-togglenumber#2]
hi, thanks for such nice package
I'm using the vim-scripts version (https://github.com/vim-scripts/togglenumber)
and I just enhance it so closing line number is also possible, so it can toggle among absolute, relative and no line number.

the PR is at https://github.com/vim-scripts/togglenumber/pull/1

any comment?

**Comment by tkhoa2711** [comment]
Any enhancements are welcomed :)

I'm just wondering that I've been able to toggle between 3 modes (including `nonumber`) without any problems so far. So `&relativenumber` and `&number` must have some kind of dependency. Let me double check.

**Comment by albinsun** [comment]
Ok, I tested it on a clean vim only with vim-scripts/togglenumber installed by vundle and found that I only can toggle between number and relativenumber.

However, Let's double check it first.

**Comment by tkhoa2711** [comment]
I'm using vim 7.3, this is a simple test that I did:

``` viml
:set nu
:echom &number &relativenumber
> 1 0
:set relativenumber
:echom &number &relativenumber
> 0 1
:set norelativenumber
:echom &number &relativenumber
> 0 0
```

This is basically the flow of logic I implemented for the plugin. Could you give this a try also?

**Comment by albinsun** [comment]
I'm using vim 7.4

``` vim
:version
> VIM - Vi IMproved 7.4 (2013 Aug 10, compiled Dec 11 2014 03:17:52)
> Included patches: 1-488
> ...
:set nu
:echom &number &relativenumber
> 1 0
:set relativenumber
:echom &number &relativenumber
> 1 1
:set norelativenumber
:echom &number &relativenumber
> 1 0
```

looks they are independent.
I also do it in 7.3, the result is the same like yours.

``` vim
:version
> VIM - Vi IMproved 7.3 (2010 Aug 15, compiled Feb 10 2013 02:28:47)
> Included patches: 1-547
> ...
:set nu
:echom &number &relativenumber
> 1 0 
:set relativenumber
:echom &number &relativenumber
> 0 1
:set norelativenumber
:echom &number &relativenumber
> 0 0
```

**Comment by tkhoa2711** [comment]
Ok, looks like vim 7.4 has made those 2 really independent. Could you please send the pull request again to this repo, the one on vim-scripts is read-only.

**Comment by tkhoa2711** [comment]
Fixed with commit 73837aec9d7442416611b3875136c398c83a608b

**Comment by albinsun** [comment]
thx
